aws_lambda_builders/workflows/nodejs_npm/workflow.py

```diff
@@ -60,35 +60,6 @@ def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtim
 
         subprocess_npm = SubprocessNpm(osutils)
 
-        self.actions = self.actions_without_bundler(
-            source_dir, artifacts_dir, scratch_dir, manifest_path, osutils, subprocess_npm
-        )
-
-    def actions_without_bundler(self, source_dir, artifacts_dir, scratch_dir, manifest_path, osutils, subprocess_npm):
```

Comment:
this was originally meant for when the workflow would have `actions_with_bundler` (for esbuild) and `actions_without_bundler`. But the bundler actions were split into a separate workflow (esbuild), so there's not much use for having `actions_without_bundler` in a separate function